### PR TITLE
Add compilation help

### PR DESCRIPTION
**Pull request by tamplan**
Use of submodule implies some commands before a compilation attempt.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,11 @@ Atril Requirements to Run Tests
 ==================================================
 * [python-dogtail](https://gitlab.com/dogtail/dogtail)
 * [python-pyatspi2](https://download.gnome.org/sources/pyatspi/)
+* git installed on your system
+
+Atril Compilation
+==================================================
+* git submodule init
+* git submodule update
```
Comment (raveit65) [inline README.md:32]: Can we use `git submodule update --remote --recursive`, please?

Comment (raveit65) [reply README.md:32]: Same for other PRs where you want to add it :)

Comment (tamplan) [reply README.md:32]: Ok, no problem.

Comment (tamplan) [reply README.md]: Done.

**Comment by tamplan** [comment]
When merged, i need to add same information for engrampa too.

**Pull request merged**